bcda/auth/client/oktaclient.go

```diff
@@ -112,23 +112,24 @@ func (oc *OktaClient) PublicKeyFor(id string) (rsa.PublicKey, bool) {
 	return key, ok
 }
 
-func (oc *OktaClient) AddClientApplication(localID string) (string, string, error) {
+func (oc *OktaClient) AddClientApplication(localID string) (string, string, string, error) {
```

Comment:
We're starting to have a confusing number of string return values.  Would it be a good idea to name them (here, in`okta.go` and in `mokta.go`)?

`func (oc *OktaClient) AddClientApplication(localID string) (clientID string, clientSecret string, clientName string, err error)`

Reply:
Now that you've gone to the trouble of naming these, I notice you've added `ClientName` to Credentials{}, which makes it a good candidate for a return type.

Reply:
I started down that path (which is why I put it in there), but that doing so ends in a method that does nothing but copy one struct into another, which did not feel idiomatic and leads down the path of refactoring how credentials are handled through the entire auth package, not just the oktaclient. I don't want to go there for this PR.